app/config/sources.inc.php

```diff
@@ -32,12 +23,107 @@
+        'public_path' => 'https://svn.mozilla.org/projects/l10n-misc/trunk/marketing/',
+        'repository'  => 'https://svn.mozilla.org/projects/l10n-misc/trunk/marketing/',
+        'vcs'         => 'svn',
+    ],
+    'about:healthreport' => [
+        'local_path'  => $repo_local_path('about:healthreport', 'fhr-l10n'),
+        'public_path' => 'https://github.com/mozilla-l10n/fhr-l10n/tree/master/',
+        'repository'  => 'https://github.com/mozilla-l10n/fhr-l10n',
+        'vcs'         => 'git',
+    ],
+    'slogans' => [
+        'local_path'  => $repo_local_path('slogans', 'slogans'),
+        'public_path' => 'https://svn.mozilla.org/projects/granary/slogans/',
+        'repository'  => 'https://svn.mozilla.org/projects/granary/slogans/',
+        'vcs'         => 'svn',
+    ],
+    'engagement' => [
+        'local_path'  => $repo_local_path('engagement', 'engagement-l10n'),
+        'public_path' => 'https://github.com/mozilla-l10n/engagement-l10n/tree/master/',
+        'repository'  => 'https://github.com/mozilla-l10n/engagement-l10n',
+        'vcs'         => 'git',
+    ],
+    'add-ons' => [
+        'local_path'  => $repo_local_path('add-ons', 'add-ons'),
+        'public_path' => 'https://svn.mozilla.org/projects/l10n-misc/trunk/add-ons/',
+        'repository'  => 'https://svn.mozilla.org/projects/l10n-misc/trunk/add-ons/',
+        'vcs'         => 'svn',
+    ],
+    'firefox-updater' => [
+        'local_path'  => $repo_local_path('firefox-updater', 'firefoxupdater'),
+        'public_path' => 'https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxupdater/',
+        'repository'  => 'https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxupdater/',
+        'vcs'         => 'svn',
+    ],
+    'firefoxos-marketing' => [
+        'local_path'  => $repo_local_path('firefoxos-marketing', 'firefoxos-marketing'),
+        'public_path' => 'https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxos-marketing/',
+        'repository'  => 'https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxos-marketing/',
+        'vcs'         => 'svn',
+    ],
+    'contribute-autoreplies' => [
+        'local_path'  => $repo_local_path('www.mozilla.org', 'mozilla_org'),
+        'public_path' => 'https://github.com/mozilla-l10n/www.mozilla.org/tree/master/',
+        'repository'  => 'https://github.com/mozilla-l10n/www.mozilla.org',
+        'vcs'         => 'git',
+    ],
+    'appstores' => [
+        'local_path'  => $repo_local_path('appstores', 'appstores'),
+        'public_path' => 'https://github.com/mozilla-l10n/appstores/tree/master/',
+        'repository'  => 'https://github.com/mozilla-l10n/appstores',
+        'vcs'         => 'git',
+    ],
+];
+
+/*
+ * If a file is not listed in $lang_flags, it's assumed to be non critical
+ * for all locales.
+ * If a flag is valid for all locales, set it to ['all']. If it's not,
+ * set the flag to the array of locales.
+ */
 $lang_flags = [];
```

Comment:
There are flags for each website ([example for mozilla.org](https://github.com/flodolo/langchecker/blob/repos/app/config/sources.inc.php#L209)), this declaration is here only to avoid warnings later when setting up the website's specific flags. Right now we're only using 'critical', 'obsolete' (to hide them), and 'opt-in'.